### ISSUE TITLE
Northstar dll crash when not supplying atleast semi valid rpak.json

### ISSUE THREAD

**Issue by EM4Volts** [R2Northstar/NorthstarLauncher#690]
To reproduce:
put rpaks in rpaks folder of mod, don't put json file in.

can be circumvented with a `rpak.json` file that isn't even valid like:

```
{
    "Preload":
    {
        "example.rpak": false
    }
}
```

(yes, it doesn't care about if that rpak even exists, putting this in resolves the crash)

**Comment by EM4Volts** [comment]
henlo frens! i would like to request to reserve this issue for myself, id like to try and fix this if possible :) 

was looking at the code rn because this issue once again happened to me today on accident and it seems like the fix could be simple.

**Comment by GeckoEidechse** [comment]
Closed by #762
